README.md

```diff
@@ -19,7 +19,7 @@ to learn how to add your device to Fleet and how to ask questions about your dev
 Fleet is [independently backed](https://linkedin.com/company/fleetdm) and actively maintained with the help of many amazing [contributors](https://github.com/fleetdm/fleet/graphs/contributors).
 
 > 📖 In keeping with our value of openness, Fleet Device Management's company handbook is public and open source.  You can read about the [history of Fleet and osquery](https://fleetdm.com/handbook/company#history) and our commitment to improving the product.
-> To upgrade from Fleet ≤3.2.0, just follow the upgrading steps for the latest release from this repository (it'll work out of the box).
+> To upgrade from Fleet ≤3.2.0, just follow the upgrading steps for the earliest subsequent major release from this repository (it'll work out of the box until the release of Fleet 5.0).
```

Comment:
Is there something special about 5.0? Following the standard upgrade instructions should work (including any major version specific instructions) if someone is on a Kolide Fleet version.

Reply:
Well I’m thinking of sails.js.  If you try to upgrade from major version 0.11 to major version 1.0, skipping 0.12 (where 0.x is technically “major” because the first number is still <1), using the migration guide from version 1.0, you dev won’t be successful.

same thing if you try to upgrade from sails 0.12 to sails 1.5, but you don’t scroll back down to the sails 1.0 release and follow those steps.


Instead, to go from 0.11 to 1.5, you need to do the 0.11 to 0.12 migration steps by looking at the 0.12 release, then do the 0.12 to 1.0 steps by looking at the 1.0 release, and then you’re good automatically for 1.5 because it’s only a minor version bump

Reply:
Ah yeah that makes sense to call out specifically to look at the breaking changes/upgrade instructions for each major version.